librz/core/core.c

```diff
@@ -1470,6 +1470,7 @@ static bool find_autocomplete(RzCore *core, RzLineCompletion *completion, RzLine
 	switch (parent->type) {
 	case RZ_CORE_AUTOCMPLT_SEEK:
 		autocomplete_functions(core, completion, p);
+		break;
```

Comment:
this is probably wrong. there should be a fallthrough.

Reply:
Why?

Reply:
because for the seek autocompletion you want *both* functions and flags to be autocompleted, AFAIK.

Reply:
hmmmmm. that kinda make sense